tool/tbot/testhelpers/srv.go

```diff
@@ -148,6 +148,27 @@ func MakeDefaultAuthClient(t *testing.T, fc *config.FileConfig) auth.ClientI {
 	client, err := authclient.Connect(context.Background(), authConfig)
 	require.NoError(t, err)
 
+	// Wait for the server to become available.
+	for i := 1; i <= 10; i++ {
+		ping, err := client.Ping(context.Background())
+		if err != nil {
+			t.Logf("server is not yet available, waiting (attempt %d of 10)...", i)
+			time.Sleep(1 * time.Second)
```

Comment:
1 whole second between attempts is quite long, perhaps we could do this every quarter?

Reply:
Easy enough, will do!